langchain/callbacks/comet_ml_callback.py

```diff
@@ -34,12 +34,10 @@ def _get_experiment(
 ) -> Any:
     comet_ml = import_comet_ml()
 
-    experiment = comet_ml.config.get_global_experiment()
```

Comment:
why get rid of this?

Reply:
The current behavior is to re-use a Comet Experiment if one was created before and if not create a new one.
That behavior can be confusing for users that creates multiple instance of `CometCallbackHandler` and can leads to subtle bugs where the new Experiment is in the wrong workspace or project_name.
So we decided to get with a more simpler approach, one instance of `CometCallbackHandler` equals one instance of Comet Experiment.

Reply:
makes sense!